website/docs/config.md

```diff
@@ -332,6 +334,35 @@ Where:
 - `{{ package }}` is the name of the package.
 - `{{ version }}` is the new version of the package.
 
+#### The `git_only` field
+
+Enable git-only mode, which determines package versions from git tags instead of the cargo registry.
+
+- If `true`, release-plz will look for existing release tags in git to determine the current
```

Comment:
```suggestion
- If `true`, release-plz will look for existing git tags to determine the current
```
Let's not introduce the concept of "release tags" since they are just git tags and there's no need to introduce a new name for them, right?